drivers/elum_huawei_smartlogger.json

```diff
@@ -0,0 +1,19 @@
+{
+  "common": {
+    "fncode": 3,
+    "words": 2,
+    "datatype": "float",
+    "order": "lsr"
+  },
+  "fields": {
+    "I_L1": {"register":  2, "unit": "A", "description": "Phase 1 current"},
+    "I_L2": {"register":  4, "unit": "A", "description": "Phase 2 current"},
+    "I_L3": {"register":  6, "unit": "A", "description": "Phase 3 current"},
+    "I_DC": {"register":  8, "unit": "A", "description": "DC current"},
```

Comment:
I think in other drivers we've used `Idc` (or similar) for the name here?

Reply:
I found dc_in_P for PV inverters at the SMA drivers. Because this meter is measuring PV inverters I followed now this format.

Reply:
Yup, makes sense!